src/suites/cts/command_buffer/copies.spec.ts

```diff
@@ -109,3 +109,67 @@ g.test('b2t2t2b', async t => {
 
   t.expectContents(dst, data);
 });
+
+g.test('imageBitmapToTexture', async t => {
+  const img = document.createElement('img');
+  img.src = '../../assets/img/test.png';
+  await img.decode();
```

Comment:
is this necessary? Shouldn't it happen implicitly during createImageBitmap, which is also async?

Reply:
Sry I'm not very farmiliar with Promise-similar thing. But for createImageBitmap, I need to get image's width and height. And I think if I don't want to implement this part in a callback. It should be await for the image been decoded.

Reply:
Oh, I understand, thanks for the explanation.
I don't think it's actually necessary to pass the width/height into createImageBitmap. Try this: `createImageBitmap(img)`, then the `img.decode()` shouldn't be needed anymore (please double check though).

Reply:
Sry, it seems I missing all your comments when I push the wrong branch.